lib/src/play_store_search_api.dart

```diff
@@ -107,33 +107,55 @@ class PlayStoreResults {
           (elm) => elm.querySelector('.wSaTQd')!.text == 'What\'s New',
           orElse: () => sectionElements[0]);
 
-      String? releaseNotes;
-
       Element? rawReleaseNotes = releaseNotesElement
           .querySelector('.PHBdkd')
           ?.querySelector('.DWPxHb');
       String? innerHtml = rawReleaseNotes!.innerHtml.toString();
+      String? releaseNotes = multilineDescription(innerHtml, rawReleaseNotes);
 
-      /// Solve the multiline problem with release notes
-      if (releaseNotesSpan.hasMatch(innerHtml)) {
-        releaseNotes =
-            releaseNotesSpan.firstMatch(innerHtml.toString())!.group(1);
-        // Detect default multiline replacement
-        releaseNotes = releaseNotes!.replaceAll('<br>', '\n');
-      } else {
-        /// Fallback to normal method
-        releaseNotes = rawReleaseNotes.text;
-      }
+      return releaseNotes;
+    } catch (e) {
+      print(
+          'upgrader: PlayStoreResults.releaseNotes exception, trying redesignedVersion: $e');
+      return redesignedReleaseNotes(response);
+    }
+  }
 
-      // print(releaseNotes);
+  /// Returns field releaseNotes from Redesigned Play Store results. When there are no
+  /// release notes, the main app description is used.
+  static String? redesignedReleaseNotes(Document response) {
+    try {
+      final sectionElements =
+          response.querySelectorAll('[itemprop="description"]');
+
+      Element? rawReleaseNotes = sectionElements.last;
+      String? innerHtml = rawReleaseNotes.innerHtml.toString();
+      String? releaseNotes = multilineDescription(innerHtml, rawReleaseNotes);
 
       return releaseNotes;
     } catch (e) {
-      print('upgrader: PlayStoreResults.releaseNotes exception: $e');
+      print('upgrader: PlayStoreResults.redesignedReleaseNotes exception: $e');
     }
     return null;
   }
 
+  static String? multilineDescription(
```

Comment:
@humanolaranja There is already a field named description and this new method `multilineDescription` is for release notes, so maybe change the name to `multilineReleaseNotes`?